submit.py

```diff
@@ -51,14 +73,27 @@
 
 print("\u2713 Data loaded")
 
-model = SpatiotemporalModel(
-    spatial_backbone=config["spatial_backbone"],
-    temporal_backbone=config["temporal_backbone"],
-    input_dim=config["input_dim"],
-    num_classes=config["num_classes"],
-    sequencelength=config["sequence_length"],
-    device=DEVICE,
-)
+if model_type == "original":
+    model = SpatiotemporalModel(
+        spatial_backbone=config["spatial_backbone"],
+        temporal_backbone=config["temporal_backbone"],
+        input_dim=config["input_dim"],
+        num_classes=config["num_classes"],
+        sequencelength=config["sequence_length"],
+        device=DEVICE,
+    )
+elif model_type == "temporal_augmentor":
+    model = TemporalAugmentor(
+        num_bands=config["input_dim"],
+        hidden_size=config["lstm_hidden_size"],
+        dropout=config["lstm_dropout"],
+        input_timesteps=config["input_timesteps"],
+        output_timesteps=config["output_timesteps"],
+        gp_inference_indexes=[config["gp_inference_index"]],
+        device=DEVICE,
+    )
+else:
+    raise NameError
```

Comment:
This is not needed, the temporal augmentor is used inside the sptiotemporalModel

Reply:
Take a look at how temporal augmentor is used in the model to see how it should be loaded

Reply:
They both have different config (e.g., config['gp_inference_index'] is not in train.py)